### ISSUE TITLE
P&D Simpson Sketchbook from Calcutta to Peshawar B2014.21.2 

### ISSUE THREAD

**Issue by edgartdata** [ycba-cia/blacklight-collections2#367]
@annabozz Simpson Sketchbook from Calcutta to Peshawar B2014.21.2 https://collections.britishart.yale.edu/catalog/tms:70162 has 29 pages. Looks like all of them have been digitized and are in the master record.

@LaurelOP What masonry image would you like?

**Comment by LaurelOP** [comment]
@edgartdata If it's possible to rotate the image, I'd suggest image 10, the page with the sketches of the man wearing a yellow hat. If rotation is not possible, then let's do image 24, the drawing of the tomb.

**Comment by annabozz** [comment]
@LaurelOP rotation is possible for the masonry image-

Which crop do you prefer- full spread or crop? I've included previews of both choices below!


Cropped:
![cropped](https://user-images.githubusercontent.com/74676099/166805608-424fd69b-0ce6-4fff-bb2d-80894ce95522.jpg)


Full spread:
![full spread](https://user-images.githubusercontent.com/74676099/166805609-ab6bffff-ecf4-4bd8-a278-7aed4b71b7ec.jpg)

**Comment by LaurelOP** [comment]
Wonderful, @annabozz ! I'd opt for cropped.

**Comment by annabozz** [comment]
- images in overall record already online
- masonry image batched for ingest
- tms records cleaned up

**Comment by annabozz** [comment]
masonry image set, NetX collection sent to P and D

Imaging work done!

**Comment by edgartdata** [comment]
Titles done.

**Comment by edgartdata** [comment]
I just unpublished the individual online page records from the online catalog.

I added the indexed keywords from the individual TMS page records to the TMS overall record for the entire sketchbook.

**Comment by annabozz** [comment]
@LaurelOP @edgartdata @mxgold 

A note to say that as per this sketchbook as an example- now that we are redoing the masonry function, I don't believe we will able to accommodate crops or rotations for the masonry view (since we're pulling from the images in the sequence, instead of a duplicate image that can be manipulated). 

Ok to proceed? See May 4th comment above for example

**Comment by edgartdata** [comment]
Good point @annabozz ! Which means that for masonry images we may have to always only choose images that can be read/right side up as they stand in the sketchbook, or be comfortable with images in the masonry view that are somewhat askew. 

In the case of this sketchbook @LaurelOP, knowing this, would you still go with this specific image even though the figures will be shown sideways?

**Comment by LaurelOP** [comment]
Many thanks @annabozz and @edgartdata. This prompted thinking and head scratching. In looking at this Simpson sketchbook, I realized that the vast majority of the images are portrait rather than landscape views. (Including, in fact, the cover.) 

I hate to generate more work, but I'm wondering about rotating all of the images, to read the sketchbook as if the spine is at the top rather than on the lefthand side. Then, only a few of the pages would require the viewer to turn their head.

If this is possible, I think we should keep the current masonry view.

Thank you--and apologies for any hassle!

**Comment by annabozz** [comment]
@LaurelOP @edgartdata 

No problem this is possible- just of note, the masonry view will be the double page spread rather than the crop if we are going to get rid of a duplicate crop- or- are we making an exception for this record?

The overall fix will require rotating images and then reingesting them into NetX, so can be done by next week.

**Comment by LaurelOP** [comment]
Thank you, @annabozz! The double-page spread is fine -- it gives a better sense of the book, even if the images are smaller.

I really appreciate your help with this.

**Comment by annabozz** [comment]
Liaised with @bobhixon and we will be able to ingest the switch out the new, rotated images after the "delete/replace" function is enabled (after the Preservica migration) so I re-opened the issues for now so we can revisit

@edgartdata @LaurelOP @mxgold I will be rotating the pub images 90 degrees clockwise as a whole- should we be rotating the bars too (even though they do not appear online?)

**Comment by LaurelOP** [comment]
Thank you, @annabozz and @bobhixon! 

Re: the bars, I'm happy either way and defer to others.

**Comment by annabozz** [comment]
@mxgold has confirmed that we just need to rotate the pubs for this one

**Comment by edgartdata** [comment]
BlackLight individual pages deleted.
